ansible_base/lib/redis/client.py

```diff
@@ -160,22 +161,27 @@ def _get_hosts(self) -> None:
             had_host_errors = False
             host_ports = self.redis_hosts.split(',')
             for host_port in host_ports:
-                try:
-                    node, port_string = host_port.split(':')
-                except ValueError:
+                response = address.classify_address(host_port)
+                if response.type == address.AddressType.UNKNOWN:
+                    logger.error(
+                        f"Specified cluster_host {host_port} is not valid; "
+                        "it is of an unknown address type and must be one of <hostname>:<port>, <ipv4>:<port> or <ipv6>:<port>"
```

Comment:
Would this be more accurate?
```suggestion
                        "it is of an unknown address type and must be one of <hostname>:<port>, <ipv4>:<port> or [<ipv6>]:<port>"
```

Reply:
It's probably better that way.  I considered adding both IPv6 but that seemed extreme.